index.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  "rules": {
+    "at-rule-empty-line-before": [ 2, "always", {
+      except: ["blockless-group"],
+    } ],
+    "at-rule-no-vendor-prefix": 0,
```

Comment:
Yes I know, it was just to make it very visibile. I disabled for such reasons https://github.com/cssrecipes/reset/blob/master/lib/button.css#L22

Even if stylelint is use before cssnext, there is some prefix in our codebase